test/integration/scene-manager-component.spec.ts

```diff
@@ -1,257 +1,114 @@
-    })
-
-    it('should return hasExtendedPermissions=true if user has deploy permission for a world', async () => {
-      mockWorldOwnerPermission.mockResolvedValue(false)
-      mockIsAdmin.mockResolvedValue(false)
-      mockWorldStreamingPermission.mockResolvedValue(false)
-      mockWorldDeployPermission.mockResolvedValue(true)
-
-      const result = await sceneManager.getUserScenePermissions(mockWorldPlace, '0xuser')
-
-      expect(result.owner).toBe(false)
-      expect(result.admin).toBe(false)
-      expect(result.hasExtendedPermissions).toBe(true)
-    })
-
-    it('should return owner=true if user is the owner of a scene', async () => {
-      mockLandPermission.mockResolvedValue({ owner: true, operator: false })
-      mockIsAdmin.mockResolvedValue(false)
-
-      const result = await sceneManager.getUserScenePermissions(mockScenePlace, '0xuser')
-
-      expect(result.owner).toBe(true)
-      expect(result.admin).toBe(false)
-      expect(result.hasExtendedPermissions).toBe(false)
-    })
-
-    it('should return admin=true if user is an admin of a scene', async () => {
-      mockLandPermission.mockResolvedValue({ owner: false, operator: false })
-      mockIsAdmin.mockResolvedValue(true)
-
-      const result = await sceneManager.getUserScenePermissions(mockScenePlace, '0xuser')
-
-      expect(result.owner).toBe(false)
-      expect(result.admin).toBe(true)
-      expect(result.hasExtendedPermissions).toBe(false)
-    })
-
-    it('should return hasExtendedPermissions=true if user is an operator of a scene', async () => {
-      mockLandPermission.mockResolvedValue({ owner: false, operator: true })
-      mockIsAdmin.mockResolvedValue(false)
-
-      const result = await sceneManager.getUserScenePermissions(mockScenePlace, '0xuser')
-
-      expect(result.owner).toBe(false)
-      expect(result.admin).toBe(false)
-      expect(result.hasExtendedPermissions).toBe(true)
-    })
-
-    it('should return all false if user has no privileges for a scene', async () => {
-      mockLandPermission.mockResolvedValue({ owner: false, operator: false })
-      mockIsAdmin.mockResolvedValue(false)
-
-      const result = await sceneManager.getUserScenePermissions(mockScenePlace, '0xuser')
-
-      expect(result.owner).toBe(false)
-      expect(result.admin).toBe(false)
-      expect(result.hasExtendedPermissions).toBe(false)
-    })
-
-    it('should return all false if user has no privileges for a world', async () => {
-      mockWorldOwnerPermission.mockResolvedValue(false)
-      mockIsAdmin.mockResolvedValue(false)
-      mockWorldStreamingPermission.mockResolvedValue(false)
-      mockWorldDeployPermission.mockResolvedValue(false)
-
-      const result = await sceneManager.getUserScenePermissions(mockWorldPlace, '0xuser')
-
-      expect(result.owner).toBe(false)
-      expect(result.admin).toBe(false)
-      expect(result.hasExtendedPermissions).toBe(false)
+    it('should return correct permissions when user is owner', async () => {
```

Comment:
I suggest we do what we did before with the text in the its, where we were describing what is the expected result instead of using the word `correct`:
```suggestion
    it('should return the owner permissions as true when user is owner', async () => {
```
The same applies for all tests here